doc/production-hosting.md

```diff
@@ -95,25 +90,11 @@ files:
 Sitting in front of the three servers is a Cloudflare Free account which
 provides several services:
 
-- Global CDN, caching, and SSL gateway for `img.shields.io`
+- Global CDN, caching, and SSL gateway for `img.shields.io` and `shields.io`
 - Analytics through the Cloudflare dashboard
-- DNS hosting for `shields.io`
```

Comment:
This bullet seems worth keeping (or clarifying?) The `shields.io` DNS is hosted on Cloudflare.

Reply:
Re-reading it, I'm not really sure why I removed that line, even though it was yesterday